### DISCUSSION TITLE
Few Update Ideas

### DISCUSSION THREAD

**Discussion by kodyVS** [devedmonton/DES-Website#272] [Ideas]
Hello DES 👋

I checked out the repo, and I see a few ways we could improve the site and give more room for growth. Also, I have a few small cleanup items I could help with. Let me know what you think of these changes.

Inconsistent relative imports
- To make moving files around easier and increase consistency, I would opt for using "@/" imports for all imports

Components imports
- Nuxt 3 comes with auto imports for components and pages. Nested folders in components files would then be named `<FolderNameComponentName>` so there is no need to have import statements and these could be removed from the pages.

Components files
- Add all the sections of pages to a "sections" sub-folder in components to split the utility components from the section components

Scroll-behavior
- When you set `scroll-behavior: smooth` in CSS it removes the question for the user whether they are on the same page as before or not. Plus, I think it looks great.

Pages and navigation:
- Place all the main page navigation under a `home` navigation link with a hover effect and then have the other nav links as new pages (all-events, resources, contact, etc). This way, you can add more content to the website easily.

**Comment by MandyMeindersma** [comment]
I absolutely love all these ideas! Would love to see them. 

Question about the last one though. 
Do you mean like you are going to add the word "home" in our menu? That means we might have to change some of the meu if it gets too squished in tablet view like so:
![image](https://github.com/devedmonton/DES-Website/assets/17459171/932b8747-cbbf-41b5-b2aa-15fbb17bd1f6)
But I might also not be understanding what you mean

**Comment by kodyVS** [comment]
Yes, but some of the other menu headers like "Get involved" would be part of the home dropdown instead of shown on the main bar. So when you hover over "Home" you will see a dropdown with "Get Involved" since it's part of the homepage and is not its own page. So there will be fewer nav-links in the nav-bar.

**Comment by MandyMeindersma** [comment]
Yeah I like that! Plus then it means we could add newsletter to the top menu after! I find the newsletter sign up gets forgotten at the bottom!
